generated/nidmm/enums.py

```diff
@@ -4,19 +4,17 @@
 
 
 class ADCCalibration(Enum):
-    AUTO = -1
+    ADC_CALIBRATION_AUTO = -1
```

Comment:
We should open an issue about this, and deal with it in a separate pool request.

Reply:
I would like to deal with it here if we can come up with a solution so that we don't have another source break.

I know the name of the attribute the enum is associated with. What if I look for that name in the value name and remove it if I find it? Do you think that would cover enough of the cases?

Reply:
Quick glance tells me that the name of the Enum won’t help much. One thing we can do is look at the values, and if they have a common prefix just remove it. But then we need to ignore PCI, RTSI, TTL… it may just work. 

Agreed on the source break, but that’s only important if we upload to PyPi. Maybe that needs to wait! Or we can make it clear in CHANGELOG that another breaker is coming.